docs/pages/docs/usage/configuration.mdx

```diff
@@ -95,29 +90,37 @@ export default getRequestConfig(async ({locale}) => {
 });
 ```
 
-To read configured messages in a component, you can use the `useMessages` hook:
+After messages are configured, they can be used via `useTranslations`.
+
+In case you require access to messages in a component, you can use a convenience API to read them from your configuration:
 
 ```tsx
+// Regular components
 import {useMessages} from 'next-intl';
-
 const messages = useMessages();
+
+// Async Server Components
+import {getMessages} from 'next-intl/server';
+const messages = await getMessages();
```

Comment:
`getMessages` might be more common than `useMessages`. This is in contrast to all other APIs from `next-intl`, like `useTranslations`/`getTranslations` etc.

Due to this, it might be a good idea to briefly mention this API on this page too. To be consistent, I've included them for the other config options below as well.